lib/penn_lib/summon_engine.rb

```diff
@@ -2,10 +2,38 @@
 class PennLib::SummonEngine < BentoSearch::SummonEngine
 
   def search_implementation(args)
+    mitigate(args)
     results = super(args)
     results.each do |item|
       item.link = 'https://proxy.library.upenn.edu/login?url=' + item.link
     end
     return results
   end
+
+  @@META_CHARS = '-|!\(\){}\[\]^~*?\\:' # '&' does not seem to be affected
```

Comment:
Generally, this sort of static information is most appropriate as a constant, e.g.: ```META_CHARS = '-|!\(\){}\[\]^~*?\\:'``` and placed at the top of the class. What is a variable that never varies?

Reply:
Ha, nice. That was indeed my intention (trying to approximate java's `public static final [...]` -- I just translated it improperly to ruby). Thanks for catching this!